Automattic/wp-calypso
# Help Center: Show in all calypso

#### Proposed Changes

* Move the Help Center from the checkout page to the layout index page.
* Show the HC in every calypso page

#### Testing Instructions



* Checkout the branch
* `yarn start` Calypso
* Be sure to be using a 10% user
* Check that the HC is showing in Calypso pages
* Remember to check on mobile
